packages/artifacts/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@aragon/osx-artifacts",
+  "version": "1.4.0-rc0",
```

Comment:
do we need rc0?
this time around we did devSepolia, so the app was using that

Reply:
Just kept what was on the other package.json, happy to edit the intended version.

Reply:
then 1.4.0

Reply:
@Rekard0 should `version` be removed from `packages/contracts/package.json` and made private?